packages/ui/client/components/FileDetails.vue

```diff
@@ -76,20 +79,32 @@ async function loadModuleGraph() {
   try {
     const gd = graphData.value
     if (!gd) {
+      loadingModuleGraph.value = false
       return
     }
 
     if (
-      !currentFilepath.value
+      force
+      || !currentFilepath.value
       || gd.filepath !== currentFilepath.value
       || (!graph.value.nodes.length && !graph.value.links.length)
     ) {
+      const moduleGraph = await client.rpc.getModuleGraph(
+        gd.projectName,
+        gd.filepath,
+        !!browserState,
+      )
+      // remove node_modules from the graph when enabled
+      if (hideNodeModules.value) {
+        moduleGraph.inlined = moduleGraph.inlined.filter(
```

Comment:
We need to clone the data when using static html reporter, the metadata is global.